src/commands/createStaticWebApp/StaticWebAppNameStep.ts

```diff
@@ -38,17 +38,21 @@ export class StaticWebAppNameStep extends AzureNameStep<IStaticWebAppWizardConte
         return !wizardContext.newStaticWebAppName;
     }
 
-    public async isRelatedNameAvailable(wizardContext: IStaticWebAppWizardContext, name: string): Promise<boolean> {
-        return await this.isSwaNameAvailable(wizardContext, name);
+    protected async isRelatedNameAvailable(wizardContext: IStaticWebAppWizardContext, name: string): Promise<boolean> {
+        if (!wizardContext.newStaticWebAppName) {
```

Comment:
I don't think this will ever be true? I checked SiteNameStep and confirmed it's only checking names for _related_ resources, not the original resource itself

https://github.com/microsoft/vscode-azuretools/blob/main/appservice/src/createAppService/SiteNameStep.ts#L76

Reply:
Oh interesting, I may have totally misunderstood this function's use.  Let me try that real quickly

Reply:
Ah, I may be using it wrong, but this is when this would be true: https://github.com/microsoft/vscode-azurestaticwebapps/pull/431/files#diff-913ec2c5bfe37c6014c9ea09a862e04a5b85eab00457ceafb2cf79a43915ca6bR27